client/rpc-spec-v2/src/transaction/api.rs

```diff
@@ -0,0 +1,38 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! API trait for transactions.
+
+use crate::transaction::event::TransactionEvent;
+use jsonrpsee::proc_macros::rpc;
+use sp_core::Bytes;
+
+#[rpc(client, server)]
+// This trait bound is required by serde, see `event`.
```

Comment:
I don't understand this comment, serde doesn't require `Clone` AFAIK.

I guess it's because the `TransactionEvent<Hash>` requires `Hash: Clone` because the `derive(Clone)?

Reply:
Yep, that's it. Have removed the comment to avoid confusion about it